src/features/game/expansion/Game.tsx

```diff
@@ -100,9 +110,36 @@ export const Game: React.FC = () => {
         </Panel>
       </Modal>
 
-      <Water level={gameState.context.state.expansions.length + 1} />
-      <Land />
-      <Hud />
+      {/* Show splash screen while session is loading  */}
+      {loadingSession ? (
+        <div className="h-screen w-full fixed top-0" style={{ zIndex: 1050 }}>
+          <Splash fadeIn={false} />
+          <Modal show centered backdrop={false}>
```

Comment:
There is a noticeable flash when showing this modal. Not a blocker, but do you have any ideas how we can prevent that?

Reply:
This is happening when the machines switch over. Might have an idea on how to fix this.

Reply:
No was unable to fix this. I can circle back at a later stage when I have more time.